src/spell.py

```diff
@@ -1,78 +1,97 @@
 #!/usr/bin/env python
 from __future__ import print_function
-
-import sys
-from typing import Dict
+import argparse
 
 law_enforcement_spelling = {
-    'a': 'Adam',
-    'b': 'Boston',
-    'c': 'Charles',
-    'd': 'David',
-    'e': 'Edward',
-    'f': 'Frank',
-    'g': 'George',
-    'h': 'Henry',
-    'i': 'Ida',
-    'j': 'James',
-    'k': 'King',
-    'l': 'Lincoln',
-    'm': 'Mary',
-    'n': 'Nora',
-    'o': 'Oscar',
-    'p': 'Paul',
-    'q': 'Queen',
-    'r': 'Robert',
-    's': 'Sam',
-    't': 'Tom',
-    'u': 'Union',
-    'v': 'Victor',
-    'w': 'William',
-    'x': 'X-Ray',
-    'y': 'Young',
-    'z': 'Zebra',
+    "a": "Adam",
+    "b": "Boston",
+    "c": "Charles",
+    "d": "David",
+    "e": "Edward",
+    "f": "Frank",
+    "g": "George",
+    "h": "Henry",
+    "i": "Ida",
+    "j": "James",
+    "k": "King",
+    "l": "Lincoln",
+    "m": "Mary",
+    "n": "Nora",
+    "o": "Oscar",
+    "p": "Paul",
+    "q": "Queen",
+    "r": "Robert",
+    "s": "Sam",
+    "t": "Tom",
+    "u": "Union",
+    "v": "Victor",
+    "w": "William",
+    "x": "X-Ray",
+    "y": "Young",
+    "z": "Zebra",
```

Comment:
Black changed the formatting from single quotes to double-quotes, I can change that back if you want.

Reply:
I guess it's fine. Let's stick to the black standard.